guide/payments/receive.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: guide
+title: Receiving bitcoin
+description: An introduction to how receiving bitcoin works.
+nav_order: 3
+parent: Payments
+permalink: /guide/payments/receive/
+main_classes: -no-top-padding
+# image: /assets/images/guide/payments/receive/header.svg
+---
+
+# Receiving bitcoin
+
+<!--
+
+Editor's notes
+
+This page should cover what to do when receiving bitcoin, how to share and copy addresses etc.
+
+- What addresses to support (native segwit by default)
+- How to present QR codes (allow for specifying an amount)
+- Generating a new address 
+
+-->
+
+For someone to receive bitcoin, the sender needs to know the recipient’s address. The receiver generates an address in their Bitcoin application and shares it with the sender as text, by scanning a [QR Code]({{ '/guide/foundations/wallet-interoperability/#qr-codes' | relative_url }}). [Payment links]({{ '/guide/foundations/wallet-interoperability/#payment-links' | relative_url }}) are also used to share additional details about a payment and can be shared in the same way.
+
+A new address should be generated for every payment, as they should only be used once. This helps safeguards the receiver's privacy.
```

Comment:
Haven't thought through this, does it also help the senders privacy?

Reply:
I would say yes. It makes the overall tracing of UTXOs more complex